docs/root/start/building/building.rst

```diff
@@ -128,3 +128,27 @@ and include the following in the WORKSPACE file::
     # Relative paths are also supported.
     path = "/somewhere/on/filesystem/envoy_build_config",
   )
+
+------------------------------
+Deploying Envoy Mobile Locally
+------------------------------
+
+~~~~~~~
+Android
+~~~~~~~
+
+To deploy Envoy Mobile's aar to your local maven repository, run the following commands::
+
+    # To build Envoy Mobile
+    bazelisk build android_dist --config=android --fat_apk_cpu=x86
```

Comment:
it might worth doing this:
```suggestion
bazelisk build android_dist --config=android --fat_apk_cpu=x86 --fat_apk_cpu=[x86|armeabi-v7a|arm64-v8a]
```

Reply:
I added a comment instead so folks can just do a copy paste without needing to edit